src/rebuild/srv.c

```diff
@@ -1230,34 +1230,38 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 			return rc;
 		}
 
-		/* Schedule reclaim to clean up current op */
+		/* Schedule reclaim to clean up current op. Let's keep go ahead to retry even if
+		 * reclaim the current rebuilding fails.
+		 */
```

Comment:
If failed to cleanup current rebuild, then next schedule will add repeated rebuild task, right? is it harmful? If that is not important, can we directly skip reclaim?

Reply:
oh, we have to delete "half rebuild" data here, otherwise if aggregation step in before next retry, it might harm the data consistency, and cause VOS complain.

Reply:
Then what the subsequent retry (after fail to RECLAIM) is for ? It seems that we do not care what caused the reclaim failure. But just as your said, what will happen if aggregation step in after reclaim failure but before next retry scheduled?